qutebrowser/config/qtargs.py

```diff
@@ -273,10 +273,21 @@ def _qtwebengine_args(
     if disabled_features:
         yield _DISABLE_FEATURES + ','.join(disabled_features)
 
-    yield from _qtwebengine_settings_args()
-
-
-_WEBENGINE_SETTINGS: Dict[str, Dict[Any, Optional[str]]] = {
+    yield from _qtwebengine_settings_args(versions, namespace, special_flags)
```

Comment:
I don't think it makes sense to pass `special_flags` here, as that's only used to handle stuff like `--enable-features` in the other code.

We also don't need `namespace` currently. I'm not sure if I'd keep it. It might make sense in the future, but then again, we might just want to leave it off if it's not needed now?